src/redux/reducers/categoryReducer.js

```diff
@@ -0,0 +1,41 @@
+import { FETCH_CATEGORY_SUCCESS, ADD_CATEGORY_SUCCESS,DELETE_CATEGORY_SUCCESS,UPDATE_CATEGORY_SUCCESS } from '../actions/type';
+
+const initialState = {
+  categories: [],
+  status: null,
+  statusType: null
+}
+
+export default function (state = initialState, action) {
+  switch (action.type) { 
+    case FETCH_CATEGORY_SUCCESS:
+      return {
+        categories: action.payload,
```

Comment:
Destructure the state instead first then int the return block

Reply:
I am sorry but I dont understand what to do

Reply:
Read other reducers to understand

Reply:
you mean using hte spread operator?